get-latest-tgf.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+get_local_tgf_version () {
+        TGF_LOCAL_VERSION=$(tgf --current-version | awk -F\  '{print $2}');
+}
+
+get_latest_tgf_version () {
+    if [ -z "$GITHUB_TOKEN" ]; then
+        echo 'Warning: GITHUB_TOKEN is not set.';
+    fi
+    TGF_LATEST_VERSION=$(curl --silent https://api.github.com/repos/coveo/tgf/releases/latest?access_token=${GITHUB_TOKEN} | grep tag_name | awk -F\" '{print $4}');
+}
+
+install_latest_tgf () {
+        curl -sL https://github.com/coveo/tgf/releases/download/v1.20.2/tgf_1.20.2_linux_64-bits.zip | gzip -d > /usr/local/bin/tgf && chmod +x /usr/local/bin/tgf;
+}
+
+if ! [ -x "$(command -v tgf)" ]; then
+  echo 'Error: tgf is not installed. Installing ...';
+  install_latest_tgf;
+  echo 'Info: Done.'
+  tgf --current-version;
+  exit 1;
```

Comment:
exit 1 means an error. Installing TGF on a machine where it is not currently installed should exit with 0 if the install was successful